lib/src/presentation/views/home/home_view.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:basic_authentication_flutter_challenge/src/domain/view/view.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/global_widgets/large_space.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/global_widgets/small_space.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/global_widgets/view_scaffold.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/views/home/home_viewmodel.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/views/home/widgets/home_logout_button.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/views/home/widgets/home_teammates_button.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/views/home/widgets/home_title.dart';
+import 'package:basic_authentication_flutter_challenge/src/presentation/views/home/widgets/home_user_details.dart';
+import 'package:flutter/material.dart';
+import 'package:stacked/stacked.dart';
+
+class HomeView extends View {
+  const HomeView({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<HomeViewModel>.reactive(
+      viewModelBuilder: () => HomeViewModel(),
+      builder: (context, model, child) {
+        return ViewScaffold(
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: const [
+                  HomeTitle(),
+                  LargeSpace(),
+                  HomeUserDetails(),
+                  LargeSpace(),
+                  HomeTeammatesButton(),
+                  SmallSpace(),
+                  HomeLogoutButton(),
+                ],
+              ),
```

Comment:
Your home widget design will cause RenderFlex overflow exception for some mobile screen sizes. You can validate your design using Widget Inspector. I faced it myself using Samsung S9.

![Screen Shot 1444-02-07 at 2 53 23 PM](https://user-images.githubusercontent.com/54819958/188269299-e59b180b-b46a-4b46-a0c1-788c09a42567.png)

Also, consider to go through the below references:
- https://stackoverflow.com/questions/49480051/flutter-dart-exceptions-caused-by-rendering-a-renderflex-overflowed
- https://github.com/flutter/flutter/blob/e3005e6962cfefbc12e7aac56576597177cc966f/packages/flutter/lib/src/widgets/basic.dart#L3724
- https://docs.flutter.dev/codelabs/layout-basics